pinot-perf/src/main/java/org/apache/pinot/perf/BenchmarkDimensionTableOverhead.java

```diff
@@ -186,9 +187,18 @@ public DimensionTableDataManager benchmark()
 
     String tableName = TABLE_NAME + "_" + _iteration;
     _tableDataManager = DimensionTableDataManager.createInstanceByTableName(tableName);
-    _tableDataManager.init(instanceDataManagerConfig, helixManager, new SegmentLocks(), tableConfig, SCHEMA,
-        new SegmentReloadSemaphore(1), Executors.newSingleThreadExecutor(), null, null, SEGMENT_OPERATIONS_THROTTLER,
-        false);
+    _tableDataManager.init(instanceDataManagerConfig,
```

Comment:
nit: was this formatting change intended? same for other files with similar changes